src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/ReleaseApprovalServiceTests.cs

```diff
@@ -1937,7 +1937,137 @@ await using (var context = InMemoryApplicationDbContext(contextId))
+                
+                result.AssertRight();
+            }
+        }
+
+        [Fact]
+        public async Task CreateReleaseStatus_HigherReview_SendsEmailIfPublicationReleaseApprover()
+        {
+            var release = new Release
+            {
+                Type = ReleaseType.AdHocStatistics,
+                Publication = new Publication { Title = "Test publication" },
+                ReleaseName = "2030",
+                Slug = "2030",
+                Version = 0,
+            };
+
+            var userPublicationReleaseApproverRole = new UserPublicationRole
+            {
+                User = new User { Id = Guid.NewGuid(), Email = "test@test.com" },
+                PublicationId = release.PublicationId,
+                Role = PublicationRole.ReleaseApprover
+            };
+
+            var contextId = Guid.NewGuid().ToString();
+            
+            await using (var context = InMemoryApplicationDbContext(contextId))
+            {
+                await context.AddRangeAsync(release, userPublicationReleaseApproverRole);
+                await context.SaveChangesAsync();
+            }
+
+            await using (var context = InMemoryApplicationDbContext(contextId))
+            {
+                var contentService = new Mock<IContentService>(MockBehavior.Strict);
+                var preReleaseUserService = new Mock<IPreReleaseUserService>(MockBehavior.Strict);
+                var userReleaseRoleService = new Mock<IUserReleaseRoleService>(MockBehavior.Strict);
+                
+                contentService.Setup(mock => mock.GetContentBlocks<HtmlBlock>(release.Id))
+                    .ReturnsAsync(new List<HtmlBlock>());
+            
+            
+                userReleaseRoleService.Setup(mock =>
+                        mock.ListUserReleaseRolesByPublication(ReleaseRole.Approver, release.Publication.Id))
+                    .ReturnsAsync(new List<UserReleaseRole>());
+                
+                var releaseService = BuildService(
+                    contentDbContext: context,
+                    contentService: contentService.Object,
+                    preReleaseUserService: preReleaseUserService.Object,
+                    userReleaseRoleService: userReleaseRoleService.Object
+                );
+                
+                var result = await releaseService
+                    .CreateReleaseStatus(
+                        release.Id, new ReleaseStatusCreateViewModel
+                        {
+                            PublishMethod = PublishMethod.Scheduled,
+                            PublishScheduled = "2051-06-30",
+                            ApprovalStatus = ReleaseApprovalStatus.HigherLevelReview,
+                            LatestInternalReleaseNote = "Test internal note",
+                            NextReleaseDate = new PartialDate { Month = "12", Year = "2077" },
+                        });
```

Comment:
Same problem as before, suspect either the email isn't being sent or there's a problem with the unit test.